nilearn/maskers/tests/test_multi_nifti_labels_masker.py

```diff
@@ -0,0 +1,321 @@
+"""
+Test the multi_nifti_labels_masker module
+"""
+
+import numpy as np
+
+import nibabel
+import pytest
+
+from nilearn._utils import testing, data_gen
+from nilearn._utils.exceptions import DimensionError
+from nilearn.image import get_data
+from nilearn.maskers import NiftiLabelsMasker, MultiNiftiLabelsMasker
+
+
+def test_multi_nifti_labels_masker():
+    # Check working of shape/affine checks
+    shape1 = (13, 11, 12)
```

Comment:
This test data generation is the same as in the tests for the `NiftiLabelsMasker`. 
Do you think we could define it as a (or multiple..??) fixture(s) in a separate file and import it in both test files? `nilearn/_utils/data_gen.py` could also be a good place to put it...